docs/services.md

```diff
@@ -1,101 +1,99 @@
+The service uses a driver-based design that let the developers develop new drivers for different backends.
+Currently, the network service supports Fabric and Orion.
+
+### Fabric Driver
+
+The Fabric driver for the network service is located in the package `token/services/network/fabric`.
+This driver assumes that a `Token Chaincode` (located in the package `token/services/network/fabric/tcc`)
+is available in the Fabric network. This chaincode provides multiple functionalities that the Fabric driver uses.
+Namely:
+- `Public Parameters Lifecycle`. The public parameters govern the behaviour of the token chaincode that means how
+  token requests are processed and translated. The list of approved issuers and auditors, and so on.
+- `Approval`. This is one of the essential steps in the lifecycle of a token transaction,
+  as we have seen in the previous section. The Token Chaincode validates the received token request and, if valid, translates 
+  it into the RW Set format understood by Fabric.
+  Notice that the Token Chaincode can only check the validity of the transaction. Though, double spending can only verified
+  at committing time. To do so, the RW Set is generated in a way to trigger MVCC conflicts in case of double spending.
+  An approval in this case is just the endorsement over the RW Set.
+
+To commit a token transaction, the Fabric network driver first derives a Fabric transaction with the RW sets 
+obtained from the approval phase. Then, this fabric transaction is sent to the Fabric ordering service.
+
+Here is the pictorial representation of the lifecycle of a token transaction for Fabric:
+
+![fabric_ttx_lifecycle.png](imgs/fabric_ttx_lifecycle.png)
+
+Let us explore now what happens when the Fabric transaction gets committed.
+We expect to see the tokens created by the transaction appearing in the `Vault` of the owners.
 The following picture shows how this happens:
 
 ![commit_process.png](imgs/commit_process.png)
 
 In more details:
-1. The FSC node listens to the `Delivery Service` events generated by its Fabric peer partner.
-   When the Fabric peer commits our token transaction `Tx`, the FSC node gets informed.
+1. Using the FSC API, each business party listens to the `Delivery Service` events generated by its Fabric peer partner.
+   When the Fabric peer commits a token transaction `Tx`, the FSC node gets informed.
 2. The `Delivery Client` informs the `Committer` that `Tx` is available.
 3. If `Tx` is deemed valid by Fabric, then the `Committer` further manipulates the RWSet by invoking all registered
-   `RWSet Processors` before the RWSet gets committed into the vault. If `Tx` is invalid, the transaction gets directly
+   `RW Set Processors` before the RW Set gets committed into the vault. If `Tx` is invalid, the transaction is
    discarded.
 4. The Token SDK installs one of these processors, the `Token RW Set Processor`. This processor extracts all the token
```

Comment:
one of these processes? Do we have more than one per driver?

Reply:
Other parts of the stack may install these processors. The application developer can do that as well.